process_new_ontology_file.py

```diff
@@ -14,7 +14,9 @@ def newest(path):
     Output: newest file in folder or None
     """
     files = os.listdir(path)
-    paths = [os.path.join(path, basename) for basename in files]
+    paths = [
+        os.path.join(path, basename) for basename in files if basename.endswith("owl")
```

Comment:
I don't think they always end in .owl 
sometimes it's just a binary file.

Reply:
The reason I added this is that after freshly cloning the repo, it wasn't the newest file in the folder. `.gitignore` is. 

1. That file should be called ".gitkeep". Pretty sure this one's on me and I actually told you to use ".gitignore"
2. Rather than specifying the file extension of what to look for, we could also just tell it to ignore `fname.startswith('.')`, which captures the current ".gitignore" case and the likely future ".gitkeep" case.

Reply:
Maybe that’s the reason why the visualization script isn’t working for Kay? She gets that the networkx object has 0 nodes and 0 edges.
Do what you think is best. The ontology file name is usually a random name when downloading from web protege.